Source/WebKit/Shared/IPCStreamTester.cpp

```diff
@@ -78,15 +78,15 @@ void IPCStreamTester::syncMessageReturningSharedMemory1(uint32_t byteCount, Comp
         auto sharedMemory = WebKit::SharedMemory::allocate(byteCount);
         if (!sharedMemory)
             return { };
-        SharedMemory::Handle handle;
-        if (!sharedMemory->createHandle(handle, SharedMemory::Protection::ReadOnly))
+        auto handle = sharedMemory->createHandle(SharedMemory::Protection::ReadOnly);
+        if (!handle)
             return { };
-        if (handle.isNull())
+        if (handle->isNull())
```

Comment:
this condition can't exist. if the MachSendRight was null, it means SharedMemory::createHandle failed